VanillaExample/RSFormViewVanillaExample/ViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ViewController.swift
+//  RSFormViewTester
+//
+//  Created by Anthony on 10/9/19.
+//  Copyright © 2019 Anthony. All rights reserved.
+//
+
+import UIKit
+import RSFormView
+
+class YourViewController: UIViewController, FormViewDelegate {
+    @IBOutlet weak var formView: FormView!;
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+    func didUpdateFields(in formView: FormView, allFieldsValid: Bool) {
+        
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        formView.delegate = self
+        
+        let birthdateField = FormField(name: "Birthdate field", // the identifier of the field, use this to collect the data later
+            initialValue: "", // the inital value of the field, if its in a date formate it will auto select that date in the picker
+            placeholder: "DOB", // The placeholder when there's no value and text field title when there is
+            fieldType: .date, //The Type of the field, .date will present a native picker view when tapping on the text field
+            isValid: false, //The initial validation state. The field won't be marked invalid until data is entered or removed
+            errorMessage: "Please enter a birthdate") //The error message to be displayed when the entry is invalid or empty
+        
+        formView.viewModel = BasicFormViewModel(items: [TextFieldCellItem(with: birthdateField)])
```

Comment:
indentation is crooked here, also we are using 2 spaces for indentation in the rest of the code base